Formula/pypy.rb

```diff
@@ -55,7 +57,17 @@ class Pypy < Formula
     sha256 "6773934e5f5fc3eaa8c5a44949b5b924fc122daa0a8aa9f80c835b4ca2a543fc"
   end
 
+  # Replace Linux tcl-tk detection with macOS section as upstream only searches system libs.
+  # The hardcoded default Homebrew "/usr/local" paths are inreplaced during install.
+  patch :DATA
+
   def install
+    # The tcl-tk library paths are hardcoded and need to be modified for non-/usr/local prefix
+    inreplace "lib_pypy/_tkinter/tklib_build.py", "/usr/local/opt/tcl-tk/", Formula["tcl-tk"].opt_prefix/""
+
+    # On macOS, `rpython/rlib/ropenssl.py` uses pkg-config to find OpenSSL
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["openssl@1.1"].opt_lib/"pkgconfig"
```

Comment:
Hopefully this will help macOS link OpenSSL. However, I can't find a way to set OpenSSL path on Linux. May need to hack up build scripts, add extra flags, or just use the bundled/static OpenSSL.